app/main/class-comment-mention.php

```diff
@@ -159,7 +162,7 @@ public function cmt_mntn_get_users( $username ) {
 	 */
 	public function cmt_mntn_at_name_filter( $content ) {
 
-		if ( function_exists( 'cmt_mntn_pro_membership_status' ) && ! cmt_mntn_pro_membership_status() ) {
+		if ( function_exists( 'cmt_mntn_pro_membership_status' ) && ! cmt_mntn_pro_membership_status() || ! cmt_mntn_check_enabled_userroles() ) {
```

Comment:
Can we make a function like "cmt_mntn_can_user_mention" and in that we can have these conditions?